docs/topics/db/models.txt

```diff
@@ -964,6 +964,35 @@ abstract base class. For example, including ``db_table`` would mean that all
 the child classes (the ones that don't specify their own :ref:`Meta <meta-options>`) would use
```

Comment:
Whilst you're editing this section, can you drop "Of course" from line 958? It's not obvious to every reader.

Reply:
NP I'll include this edit as well.